### PR TITLE
Support to OS Deployment Server 

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Support to OS Deployment Server 

### Issues Resolved
N/a

### Check List
- [x] New functionality includes sanity testing.
  - [x] All sanity tests pass. (`$ ansible-test sanity`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

File: roles/oneview_os_deployment_server/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+| state  |  Yes |  | <ul> <li>present</li>  <li>absent</li>  </ul> |  Indicates the desired state for the resource. `present` ensures data properties are compliant with OneView. `absent` removes the resource from OneView, if it exists.
+| validate_etag  |   |  True  | <ul> <li>true</li>  <li>false</li> </ul> |  When the ETag Validation is enabled, the request will be conditionally processed only if the current ETag for the resource matches the ETag provided in the data.  |
+
+
+## Example Playbook
+
+```yaml
+- hosts: all
+  collections:
+    - name: hpe.oneview
```
Comment (yuvirani) [inline roles/oneview_os_deployment_server/README.md:26]: Change to hpe.oneview

File: roles/oneview_os_deployment_server/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+galaxy_info:
+  role_name: oneview_os_deployment_server_facts
```
Comment (yuvirani) [inline roles/oneview_os_deployment_server/meta/main.yml:3]: role name should be oneview_os_deployment_server

File: tests/unit/test_oneview_os_deployment_server.py [hunk 3]
```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2017) Hewlett Packard Enterprise Development LP
```
Comment (yuvirani) [inline tests/unit/test_oneview_os_deployment_server.py:4]: Change to 2020

File: plugins/modules/oneview_os_deployment_server.py [hunk 4]
```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2017) Hewlett Packard Enterprise Development LP
```
Comment (yuvirani) [inline plugins/modules/oneview_os_deployment_server.py:4]: Change to 2020

Comment (AsisBagga) [reply plugins/modules/oneview_os_deployment_server.py:4]: done

File: tests/unit/test_oneview_os_deployment_server.py (hunk 3 above)
Comment (AsisBagga) [reply tests/unit/test_oneview_os_deployment_server.py:4]: done

File: roles/oneview_os_deployment_server/meta/main.yml (hunk 2 above)
Comment (AsisBagga) [reply roles/oneview_os_deployment_server/meta/main.yml:3]: done

File: roles/oneview_os_deployment_server/README.md (hunk 1 above)
Comment (AsisBagga) [reply roles/oneview_os_deployment_server/README.md:26]: its the same

**Pull request merged**